src/integrationTest/resources/caseworker-issue-application-about-to-submit-solicitor-service-response.json

```diff
@@ -0,0 +1,125 @@
+{
+  "data": {
+    "divorceOrDissolution": "divorce",
+    "applicant1FirstName": "test_first_name",
+    "applicant1MiddleName": "test_middle_name",
+    "applicant1LastName": "test_last_name",
+    "applicant1Email": "test@test.com",
+    "applicant1AgreedToReceiveEmails": null,
+    "applicant1LanguagePreferenceWelsh": "No",
+    "applicant1NameDifferentToMarriageCertificate": null,
+    "applicant1LastNameChangedWhenMarried": null,
+    "applicant1NameChangedHow": null,
+    "applicant1NameChangedHowOtherDetails": null,
+    "applicant1PhoneNumber": null,
+    "applicant1ContactDetailsConfidential": "share",
+    "applicant1Gender": "female",
+    "applicant1CorrespondenceAddress": null,
+    "applicant1SolicitorRepresented": null,
```

Comment:
you can remove all fields and during assertion add config `TREATING_NULL_AS_ABSENT`